en/app.ftl

```diff
@@ -821,9 +821,9 @@ multi-part-onboarding-reply-headline = Reply to your emails
 
 multi-part-onboarding-premium-welcome-description-2 = No more five-mask limit: you can now generate as many custom or random masks as you need. On desktop, you can use the { -brand-name-relay } add-on to create them on the go.
 # Variables:
-#   $mozmail (string): domain used by Relay masks (.mozmail.com)
-multi-part-onboarding-premium-get-domain-description-3 = With a custom subdomain, you can create masks without having to generate them beforehand.
-    Need one to sign up for a newsletter? Just say “read@customsubdomain{ $mozmail }”
+#   $mozmail (string): domain used by Relay masks (mozmail.com)
+multi-part-onboarding-premium-get-domain-description-4 = With a custom subdomain, you can create masks without having to generate them beforehand.
```

Comment:
I meant to say -3b or -3-b as it is the 3rd paragraph or bullet so it doesn't look like we missed a paragraph and jump to -4. I am looking at the content before this string.

Reply:
Oh, so do you meant you want the variable name to be something like `multi-part-onboarding-premium-get-domain-description-3b`?

Reply:
Yes, basically it is a replacement of an older version of the same string, either due to minor content update or bug fix.

Reply:
Done!